css/stylesheet.css

```
@@ -153,4 +153,54 @@ p span {
     line-height: 20px;
     font-weight: bold;
     transform: translate( 0, -120px);
-}
\ No newline at end of file
+}
+
+.introduce {
+    margin: 100px auto;
+    width: 90%; 
+}
+
+.section-headline span {
+    color: #426579;
+    width: max-content;
+    margin: 0 auto;
+    position: relative;
+    display: block;
+    text-align: center;
+}
+
+.section-headline span::after {
+	display: inline;
+	content: "";
+	width: 100%;
+	height: 10px;
+	background-color: #b0c4de;
+	position:absolute;
+	bottom: -10px;
+	left: 0;
+	border-radius: 10px;
+	z-index: 100; 
+}
```

IMO
position: absoluteがなくても実装できると思いました。
absoluteは要素の高さがなくなってしまうなど制御が大変なので、できるだけ使わずシンプルに実装することを心がけています！

上記で`<br>`を無くしたので、display: block;にして余白は隣接要素を使用しました。


Q
z-indexは必要でしょうか？

```suggestion
.section-headline {
    color: #426579;
    width: max-content;
    margin: 0 auto;
    display: block;
}

.section-headline::after {
    display: block;
    content: "";
    height: 10px;
    background-color: #b0c4de;
    border-radius: 10px;
}

.section-headline + .section-headline {
    margin-top: 10px; /* すみません、適当です */
}
```

---

参考コードも細かく教えていただいてありがとうございます。
隣接要素を初めて知りました、いつもとても勉強になります！
修正いたしました！

707ca54